keyboards/hubble/hubble.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Also, considering that the info.json has matrix information, this file (along with the hubble.c file) can and should be removed, completely.

Reply:
removed